beacon-chain/execution/rpc_connection.go

```diff
@@ -105,46 +104,16 @@ func (s *Service) retryExecutionClientConnection(ctx context.Context, err error)
 // is ready to serve we connect to it again. This method is only
 // relevant if we are on our backup endpoint.
 func (s *Service) checkDefaultEndpoint(ctx context.Context) {
```

Comment:
We only have 1 endpoint now, this will basically cause prysm to keep on creating new connections to our execution node. This method should be removed since it serves no purpose with 0 fallback endpoints.